### DISCUSSION TITLE
systemd service unit file missing in package

### DISCUSSION THREAD

**Discussion by CleyFaye** [verdaccio/verdaccio#3536] [Q&A] [answered]
Hi,

We're running verdaccio on our servers using systemd, as described in the documentation (https://verdaccio.org/docs/server-configuration/#using-systemd)

We recently updated, and the verdaccio.systemd file is now missing from the package we get from npm (the whole systemd directory is missing).

We saw nothing about this in the changelog, so the question is, is this an intended change? I think it is fine to grab the file by hand from the repository instead of bundling it with the npm package, but it was also convenient having it there. Also if it is expected to not be bundled in the future, the documentation may need an upgrade in that regard.

**Comment by juanpicado** [comment, marked as answer]
Ok, I'll include it in the next release :) I didn't have until now any feedback that was actually being used.

**Comment by CleyFaye** [comment]
Nice, thanks :)

**Reply by juanpicado** [reply]
https://github.com/verdaccio/verdaccio/pull/3556
